packages/angular/.storybook/preview.js

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (c) 2016-2020 VMware, Inc.
+ * All Rights ReserveThis software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+import '@clr/icons';
+import { addDecorator, addParameters } from '@storybook/angular';
+import { withCssResources } from '@storybook/addon-cssresources';
+import darkTheme from '../dist/clr-ui/clr-ui-dark.min.css';
```

Comment:
I think this should be `../../clr-ui/clr-ui-dark.min.css`
`npm run build-storybook` fails for me with 
```
Error: Can't resolve '../dist/clr-ui/clr-ui-dark.min.css'
```
I guess it's related.

Reply:
Thanks for pointing this out. I hadn't given the build command any work yet. 
I updated the build-command to handle building the css before it builds storybook, like it does when run for development.